bin/dbsbuffer-file-fix.py

```diff
@@ -9,7 +9,8 @@
 from WMCore.WMInit import connectToDB
 
 def fixDBSmissingFileAssoc():
-    os.environ['WMAGENT_CONFIG'] = '/data/srv/wmagent/current/config/wmagent/config.py'
+    os.environ['WMAGENT_CONFIG'] = '/data/srv/wmagent/current/config/config.py'
```

Comment:
Andrea, I meant to say that we need to update the environment variable and assign a default value as well. So the current code is still missing the new env var, please change this line to:
```
    os.environ['WMA_CONFIG_FILE'] = '/data/srv/wmagent/current/config/config.py'
```
and squash commits.